src/main.rs

```diff
@@ -24,11 +24,11 @@ use youki::commands::start;
 use youki::commands::state;
 use youki::rootless::should_use_rootless;
 
-/// High-level commandline option definition
-/// This takes global options as well as individual commands as specified in [OCI runtime-spec](https://github.com/opencontainers/runtime-spec/blob/master/runtime.md)
-/// Also check [runc commandline documentation](https://github.com/opencontainers/runc/blob/master/man/runc.8.md) for more explanation
+// High-level commandline option definition
```

Comment:
This is to prevent it from being output by the help command.

Reply:
lgtm :+1: